app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    result = split_integer(6, 2)
+    assert sum(result) == 6
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    result = split_integer(6, 2)
+    assert set(result) == {3}
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    result = split_integer(6, 1)
+    assert result == [6]
```

Comment:
Code Style issue: You should not create unnecessary variables for simple tests. You can directly use the function call in the assert statement. Example: `assert split_integer(6, 1) == [6]`